gnews_api.py

```diff
@@ -0,0 +1,6 @@
+import requests
+
+
+def fetch_news(query):
+    response = requests.get(f'https://gnews.io/api/v4/search?q={query}&token=API_KEY')
```

Comment:
Where will this key be loaded from?

Reply:
The API key will be loaded from an environment variable. This is a common practice to avoid hardcoding sensitive information like API keys into the code. I will update the code to load the API key from an environment variable using the `os` library.